starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +100,13 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(wordsUnique){
+  var uniqueNames = [];
+  
```

Comment:
you were on the right path here. You just ha to finish the function with something like this 
`
  array.map (function (word) {
    if (newArray.indexOf(word) == -1)
      newArray.push(word);
  });

  return newArray;
}
`